scripts/ci/update_server_names.py

```diff
@@ -0,0 +1,156 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import re
+import requests
+import sys
+import xml.etree.ElementTree as etree
+from translate.misc.xml_helpers import reindent
+
+MULLVAD_SERVER_LIST_API = "https://api.mullvad.net/public/relays/wireguard/v2"
+
+def fetch_server_list():
+    country_names = []
+    city_names = []
+
+    try:
+        response = requests.get(MULLVAD_SERVER_LIST_API)
+
+        if response.status_code == 200:
+            data = response.json()
+
+            locations = data.get('locations', None)
+            if locations:
+                for country_code, location in locations.items():
+                    country_code = country_code.split("-")[0]
+
+                    country_name = location.get("country", None)
+
+                    # We don't want duplicates
+                    # so let's check if this country code is in the list already
+                    found_country = [c for c in country_names if c.get("code") == country_code]
```

Comment:
Why make `country_names` a list of dictionary instead of just a dictionary?

```python
            locations = data.get('locations', {})
            for location_code, location in locations.items():
                country_code = location_code.split("-")[0]
                country_name = location.get("country", None)
                if country_name and country_code not in country_names:
                    country_names[country_code] = country_name

                city = location.get("city", None)
                if city and city not in city_names:
                    city_names.append(city)
```

And later

```python
for country_code, country_name in countries.items():
        string_map[country_code] = country_name
```

Reply:
> Why make country_names a list of dictionary instead of just a dictionary?

No good reason other than I did not think too much about it. Will change, making it a just a dict makes more sense.